apps/mg/src/mg_pulse.erl

```diff
@@ -61,6 +61,9 @@
     % Обслуживание обработчиков машин
     | #mg_worker_call_attempt{}
     | #mg_worker_start_attempt{}
+    % Обслуживание хранилища
```

Comment:
Since you are touching this module anyway, I would suggest changing all comments to English (for consistency). :cat2:

Reply:
:gb:

Reply:
![image](https://user-images.githubusercontent.com/1768466/74148706-59e64780-4c06-11ea-8964-68645d7d080b.png)